pom.xml

```diff
@@ -209,10 +209,10 @@
         <httpmime.version>${httpclient.version}</httpmime.version>
         <jai-imageio-core.version>1.3.1</jai-imageio-core.version>
         <JavaAPIforKml.version>2.2.1_CODICE_1</JavaAPIforKml.version>
-        <jackson.version>2.9.10</jackson.version>
+        <jackson.version>2.10.0</jackson.version>
         <!-- databind 2.9.10.1 fixes CVE-2019-16942, CVE-2019-16943, CVE-2019-17531.
         This separate version can be removed when jackson.version is upgraded to a version newer than 2.9.10. -->
-        <jackson.databind.version>2.9.10.1</jackson.databind.version>
+        <jackson.databind.version>2.10.0</jackson.databind.version>
```

Comment:
If these are matchingnow (jackson and jackson databind) you should remove the comment and refactor all instances of jackson.databind.version -> jackson.version